### ISSUE TITLE
Preview container is missing role="button" and aria-label

### ISSUE THREAD

**Issue by karendolan** [polimediaupv/paella-core#299]
PreviewContainer is a button that plays the video and is missing the role and aria label that identifies this to screen readers.

https://github.com/polimediaupv/paella-core/blob/main/src/js/core/PreviewContainer.js#L43-L63

**Comment by karendolan** [comment]
Sorry, this was  a duplicate ticket from https://github.com/polimediaupv/paella-core/issues/295

**Comment by karendolan** [comment]
Reference https://github.com/polimediaupv/paella-core/issues/295
